docs/index.md

```diff
@@ -8,98 +8,138 @@ site:
 
 +++ {"class": "col-page-inset"}
 
+:::{important} Jupyter Book 2 is here!
```

Comment:
This box feels a bit weird, the reasons I can really put it into words though, but maybe it's just the colour blue is what's weird.

How would feel about having it as an `:::{attention}` instead, that may work better with the loudspeaker and yellow.